index.bs

```diff
@@ -770,7 +753,7 @@ console.log(&#96;values: ${bufferE}&#96;);
 </div>
 
 ### Asynchronous Execution ### {#api-mlcontext-async-execution}
-Asynchronously carries out the computational workload of a compiled graph {{MLGraph}} on a separate timeline, either on a worker thread for the CPU execution, or on a GPU timeline for the submission of GPU workload on the command queue. The asynchronous nature of this call avoids blocking the calling thread while the computation for result is ongoing. This method of execution requires an {{MLContext}} created with {{MLContextOptions}}. Otherwise, it throws an "{{OperationError}}" {{DOMException}}.
+Asynchronously carries out the computational workload of a compiled graph {{MLGraph}} on a separate timeline, either on a CPU worker thread or on a GPU timeline submitting a GPU workload on the command queue. The asynchronous nature of this call avoids blocking the calling thread while the computation for result is ongoing. This method of execution requires an {{MLContext}} created with {{MLContextOptions}}. Otherwise, it throws an "{{OperationError}}" {{DOMException}}.
```

Comment:
@wchao1115 

> As for the OperationError failure cases for `compute` I think it's overkill. I don't recall why we need that. We should remove those restrictions.

Do you mean it should allow the webgpu context supports the `MLContext.compute()` execution method? However, this is not reflected by this PR because the spec text still enforces a default context. You may want to clarify that. Thanks.